### DISCUSSION TITLE
Logarithmic scale option for the Y-axis of a Histogram

### DISCUSSION THREAD

**Discussion by badcc** [grafana/grafana#69756] [Dashboards, Panels & Visualizations]
I was surprised to find that there is not a logarithmic option for the Y-axis of a Histogram chart. I have many use cases for this, such as for displaying things with great monotonic inequality.

Is there some reason this was left off? In an older discussion of adding to the [X-axis](https://github.com/grafana/grafana/issues/35002) I see in their example photo that it actually looks like this feature USED to exist..

E.g. 
![image](https://github.com/grafana/grafana/assets/990820/d2dd6e75-340e-4b95-a404-a0bb82f32aef)

**Comment by leeoniya** [comment]
i guess one issue is the inability to show 0 on a logarithmic scale, and histograms show buckets of counts, where counts are frequently 0. i can't really find any examples online. we could add symlog (technically, inverse hyperbolic sine), which becomes linear below some chosen threshold, and can go to 0.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82659. Feel free to continue the discussion around this there. Thank you!
